### PR TITLE
feat: test markets for taker and maker fees & some small docker fixes

### PR DESCRIPTION
**Pull request by lead4good**

### CODE REVIEW

File: doc/manual.rst [hunk 1]
```diff
@@ -3479,11 +3479,37 @@ Accessing trading fee rates should be done via the ``.markets`` property, like s
 
 .. code:: javascript
 
-   exchange.markets['ETH/BTC']['taker'] // taker fee rate for ETH/BTC
    exchange.markets['BTC/USD']['maker'] // maker fee rate for BTC/USD
 
+The ``.markets`` property may contain additional fee related information:
```
Comment (kroitor) [inline doc/manual.rst:3484]: Can you plz move these edits to wiki/Manual.md ? The .rst file is transpiled from .md: 

- https://github.com/ccxt/ccxt/blob/master/CONTRIBUTING.md#how-to-contribute-code
- https://github.com/ccxt/ccxt/blob/master/CONTRIBUTING.md#transpiled-generated-files

Thanks!

File: wiki/Manual.md [hunk 2]
```diff
@@ -3226,12 +3226,39 @@ The `calculateFee` method will return a unified fee structure with precalculated
 Accessing trading fee rates should be done via the `.markets` property, like so:
 
 ```Javascript
-exchange.markets['ETH/BTC']['taker'] // taker fee rate for ETH/BTC
```
[the remark this answers is not in the document]
Comment (lead4good) [reply wiki/Manual.md]: I thought it was enough to show how to access the `.markets` property once and then mention the attributes in the structure down below, but I'm fine with readding it :)

**Comment by lead4good** [comment]
@kroitor I understand your fix, the initial `fetchMarkets test` only ran the exchange provided `fetchMarkets` function without setting the fees which happens in `loadMarketsHelper`. So my tests all failed for those exchanges which cannot dynamically load the fees in fetchMarkets, but might have fees statically defined.
One suggestion: The test could allow taker and maker as undefined for exchanges which have `fetchFees` or `fetchTradingFees` functionality, for all other exchanges it should be a number?

**Comment by kroitor** [comment]
> One suggestion: The test could allow taker and maker as undefined for exchanges which have fetchFees or fetchTradingFees functionality, for all other exchanges it should be a number?

We want `test.market.js` to test just the format of the unified market structure, for the separation of concerns. It should check a market, but it should be agnostic of the methods used to fetch markets. For that matter, we should not mention `fetchMarkets`, `fetchFees`, `fetchTradingFees` or any other method in `test.market.js` because `test.market.js` is meant to check the market structure, not some method. The market structure should be a hard format and the test has to make sure that the format of the market is the same across all exchanges. In other words, it should check whether all fields of a market are there, regardless of the loading sequence. If we want to later check for dynamic vs hardcoded fees – that has to be done in `test.fetchMarkets.js`, or `test.loadMarkets.js`, or `test.fetchFees.js` or `test.fetchTradingFees`. Thus in `test.market.js` we check if `taker` and `maker` are there, and they have the allowed values (undefined or float).

As you know, we have exchange-wide `exchange.fees` for hardcoded fees, and each market is extended with those fees. So, in order to distinguish hardcoded vs dynamic in the further tests, we can add a `dynamicFees: false` to `exchange.fees`, and set `dynamicFees: true` for the exchanges that serve fees in markets.

Let me know if that makes sense or if I am missing something. Going to merge it today! Thank you!

**Pull request merged**